frappe/__init__.py

```diff
@@ -2308,6 +2308,20 @@ def publish_realtime(*args, **kwargs):
 	return frappe.realtime.publish_realtime(*args, **kwargs)
 
 
+def publish_task_progress(message: str, progress: float):
+	"""
+	Publish task progress to the user
+	:param message: Message for the user
+	:return: Nothing
+	"""
+	from frappe.core.doctype.background_task.background_task import publish_task_progress
```

Comment:
If we want to make this an API (and other functions in the same file) it should probably be moved / re-exported somewhere else.